Jenkinsfile.conda

```diff
@@ -0,0 +1,3 @@
+
+@Library('JenkinsShared')_
+CondaPipeline(["configuration_repo"], "ts-nightreport", "lsst.ts.nightreport", "noarch")
```

Comment:
What is `configuration_repo`? Is that the GitHub repository?

Reply:
My fault, missed to remove this from the template. I'll make the change :+1:.